assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java

```diff
@@ -1,366 +0,0 @@
-/*
```

Comment:
Why was this file deleted?

Reply:
Good catch, it's part of a manual test added for the update dialog node nullable feature